README.md

```diff
@@ -1,61 +1,123 @@
-# Furyagent v0.2.3
+<h1 align="center">
+  <img src="docs/assets/furyagent.png" width="200px"/><br/>
+  Furyagent
+</h1>
+
+<p align="center">Automate some of the Kubernetes cluster management burocracy</p>
 
 [![Build Status](http://ci.sighup.io/api/badges/sighupio/furyagent/status.svg?ref=refs/tags/v0.2.3)](http://ci.sighup.io/sighupio/furyagent)
+![Release](https://img.shields.io/github/v/release/sighupio/furyagent?label=Release)
+![License](https://img.shields.io/github/license/sighupio/furyagent)
+[![Slack](https://img.shields.io/badge/slack-@kubernetes/fury-yellow.svg?logo=slack&label=Slack)](https://kubernetes.slack.com/archives/C0154HYTAQH)
 
-## Install
+<!-- <KFD-DOCS> -->
 
-You can find `furyagent` binaries on the [Releases page](https://github.com/sighupio/furyagent/releases).
+**Furyagent** is a simple CLI tool to automate some common tasks in a Kubernetes cluster management.
 
-Supported architectures are (64 bit):
+Furyagent can help you to:
 
--   `linux`
--   `darwin`
+1. Backup and restore etcd
+2. Dispatch etcd and master certificates
+3. Manage OpenVPN access
+4. Manage SSH access
 
-Download right binary for your architecture and add it to your PATH. Assuming it's downloaded in your
-`~/Downloads` folder, you can run following commands (replacing `{arch}` with your architecture):
+## Installation
 
-```shell
-chmod +x  ~/Downloads/furyagent-{arch}-amd64 && mv ~/Downloads/furyagent-{arch}-amd64 /usr/local/bin/furyagent
+You can find `furyagent` binaries on the [Releases page](https://github.com/sighupio/furyagent/releases).
+
+To download the latest release, run:
```

Comment:
To download the latest release directly from the command line, run:

Reply:
✅